content/source/docs/enterprise/private/automating-the-installer.html.md

```diff
@@ -292,6 +292,13 @@ while ! curl -ksfS --connect-timeout 5 https://tfe.example.com/_health_check; do
 done
 ```
 
+## If the installation does not appear to be configured correctly
+
+If the installation is successful, but prompts for additional configuration rather than skipping that step, it's likely that the supplied configuration files were not applied during installation.
+
+* Verify the locations of the files: `/etc/replicated.conf` for the Replicated configuration file, and the path specified in the Replicated configuration file for the application settings. 
```

Comment:
```suggestion
* Verify the locations and permissions of the files: `/etc/replicated.conf` for the Replicated configuration file, and the path specified in the Replicated configuration file for the application settings. 
```

Reply:
Great suggestion - what are the correct permissions? 600? 400? 644?

Reply:
644 will work fine. If we want tighten permissions for security puproses then it's 600 and owned by the replicated user